actors/src/main/kotlin/ktx/actors/events.kt

```diff
@@ -88,6 +89,20 @@ inline fun <T : Actor> T.onClickEvent(
   return clickListener
 }
 
+/**
+ * Attaches a [ClickListener] to this tree.
+ * @param lsitener invoked each time a node in this tree is clicked. Consumes the triggered [InputEvent] and the
+ * [Tree] the listener was originally attached to as `this`. The received Node is the node that was clicked.
+ * @return [ClickListener] instance.
+ */
+inline fun <T : Tree<*, *>> T.onItemClick(crossinline listener: ((Tree.Node<*, *, *>) -> Unit)): ClickListener {
+  val clickListener = object : ClickListener() {
+    override fun clicked(event: InputEvent?, x: Float, y: Float) = listener(selectedNode)
```

Comment:
`getSelectedNode` returns only the first selected node, and can return null. It might be more appropriate to return the entire `Selection` object or a list of selected nodes returned by it.